_posts/en/newsletters/2019-06-12-newsletter.md

```diff
@@ -0,0 +1,328 @@
+---
+title: 'Bitcoin Optech Newsletter #50'
+permalink: /en/newsletters/2019/06/12/
+name: 2019-06-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes meetings from the CoreDev.tech event,
+describes a proposed amendment to BIP125 replace-by-fee, links to an
+Optech executive briefing video about Schnorr/Taproot, and briefly
+celebrates the 50th weekly issue of the Optech Newsletter.  Also
+included are our regular sections on bech32 sending support and notable
+changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Bitcoin Core contributor meetings:** many contributors to Bitcoin Core
+  and LN met in person for a periodic [CoreDev.tech][] event before
+  last weekend's Breaking Bitcoin conference in Amsterdam.  Transcripts
+  of several of the meetings were provided by contributor Bryan Bishop
+  to his [page for the meeting series][coredev.tech transcripts] that
+  also includes transcripts from previous events.  The transcripts,
+  which were typed real-time, don't attempt to attribute statements to
+  specific speakers but instead capture the essential technical content
+  of the meeting for contributors who couldn't attend as well as for
+  anyone interested in Bitcoin Core development.  All the discussions
+  were great, so we felt compelled to give each a small introduction:
+
+    - June 5th: a session on [code review][xs review] discussed results
```

Comment:
I really did like all of these transcripts, so I would like to mention each of them if possible.  As a possible compromise to address your concerns about length, I've pushed 6feb4ee141d03a5c776be7c8b2e0bfb5a23261f6 that halves the word count of this item, from 747 words to 373 (source code words, including link identifiers).

I think you have a point about internal Bitcoin Core project discussions not being of network-wide importance (unless things go horribly wrong), but I suspect that they're more popular than you suspect.  People are always interested in how successful projects are run and how they deal with the same problems every large-scale group effort has.  Although particular details like `IsMine()` are irrelevant to anyone not hip-deep in bitcoind internals, I think we've mentioned things like descriptors positively enough times that some readers might be interested in getting a glimpse of how much work it might actually require to convert an assumed-script wallet to a descriptor wallet.